src/workers/cdc.ts

```diff
@@ -488,6 +490,12 @@ const onSourceChange = async (
   logger: FastifyBaseLogger,
   data: ChangeMessage<Source>,
 ) => {
+  if (data.payload.op === 'c') {
+    if (data.payload.after.type === SourceType.Squad) {
+      await notifySourceSquadCreated(logger, data.payload.after);
```

Comment:
Shouldn't we just notify on every source created and then filter on the read side?

Reply:
Thought about it, was not sure on the cost of publishing a bunch of messages that we do not use currently. In that case I would just create `api.source-created` topic and then separate subscription that handles only squad stuff. Would that be better approach? Never worked with our topics/subscriptions before so generally interested on what would we prefer.

Reply:
Since creation of source doesn't happen so often compared to squads I think we should be ok. Your line of thought is fine, cost should be taken into account but it's minor in this case. But I also think about not having to introduce new topics for other use cases in the future. Just like with posts we have a single object and not topic per post type. This is how I see it.

Reply:
ah yes this is my suggestion as we only want to act on squads for now I thought it would be more efficient to only listen to that and act on those.
To save bandwidth.

Might be good to have a chat about this and what we prefer (or document it)